docs/install/ubuntu_setup.md

```diff
@@ -170,44 +191,72 @@ If building on CPU and using OpenBLAS:
 ```bash
     git clone --recursive https://github.com/apache/incubator-mxnet.git
     cd incubator-mxnet
-    echo "USE_OPENCV = 1" >> ./config.mk
-    echo "USE_BLAS = openblas" >> ./config.mk
-    make -j $(nproc)
+```
+
```

Comment:
Should put something here instead of having two code blocks back to back... some description...